packages/@yodaos/task/app-task.js

```diff
@@ -0,0 +1,91 @@
+'use strict'
+
+var AtomicTask = require('./atomic-task').AtomicTask
+var MediaPlayer = require('@yoda/multimedia').MediaPlayer
+var tts = require('@yodaos/speech-synthesis').speechSynthesis
+
+/**
+ * @description App task class is a wrapper of atomic task class which provide much easier usage by only define several task type and resource value.
+ * @memberof module:`@yodaos/task`
+ * @param {Array} tasks The quark tasks description object array.
+ * @param {string} [name] The task name.
+ * @throws `TypeError` if arguments are not qualified.
+ * @example
+ ```
+  // An atomic app task example which include 4 quark tasks: first speak a text, then play a piece of ringtone and then noop for a while and play ringtone again at last.
+  var AppTask = require('@yodaos/task').AppTask
+  var task = new AppTask([
+      { tts: '你的计时到了。' },
+      { media: 'system://alarm_default_ringtone.mp3' },
+      { timeout: 4000 },
+      { media: 'system://alarm_default_ringtone.mp3' }
+    ], 'timer-timeup-task')
```

Comment:
What's the difference between with the VoiceFlow at #652?

Reply:
The basic purpose is the same, but the internal implementation logic and the interaction with the caller, as well as the exposed interface, are very different. 
You can check the usage example for detail.

Reply:
Moved to @yodaos/application/vui, together with #652 .